### ISSUE TITLE
Can't overwrite key color

### ISSUE THREAD

**Issue by sarahbethfederman** [mrmlnc/vscode-json5#2]
I'm new to this, but I'm having trouble overwriting the key color.
I'm using the insiders build Version 1.20.0-insider 

Quoted keys seem to not register under the "string.key.json" scope. Unquoted keys do. Is this intentional? Normal JSON syntax highlighting doesn't seem to have this issue.

Here's what is in my user settings:
```
"editor.tokenColorCustomizations": {
    "textMateRules": [
      {
        "scope": "string.key.json5",
        "settings": {
          "foreground": "#8BDEDB"
        }
      }
    ]
}
```

Here is the result:
![screen shot 2018-01-08 at 12 49 49 pm](https://user-images.githubusercontent.com/5673114/34691837-c3d0820c-f472-11e7-84df-5b848d4bd3be.png)

And unquoted: 
![screen shot 2018-01-08 at 12 53 32 pm](https://user-images.githubusercontent.com/5673114/34691885-fa219b8e-f472-11e7-9d77-3995440b0cea.png)

**Comment by msgctcheng** [comment]
This still is not supported to my knowledge; if anyone is still working on this repo.

**Comment by kriszcson** [comment]
"textMateRules": [
            {
                "scope": "string.json",
                "settings": {
                    "foreground": "#8BDEDB"
                }
            }
        ]



This changing the key of the pairs

**Comment by Avasam** [comment]
Yeah doesn't work for me either. Is there also a way to make the numbers green again like the default json token color? (instead of the current white).

Edit: I started using https://github.com/katjanakosic/vscode-json5 instead, which forks this repo.
